CHANGELOG.rst

```diff
@@ -21,6 +21,13 @@ Removed
 Fixed
 -----
 
+[1.0.10] - 2019-06-??
+^^^^^^^^^^^^^^^^^^^^^
+
+Fixed
+-----
+- error when using mysql tracker store
```

Comment:
please add this to the `unreleased` section, as `1.0.10` is not tagged

Reply:
Moved it!

Reply:
it would be great to be a bit more descriptive here, e.g.
```suggestion
- SQL ``data`` column type to ``Text`` for compatibility with MySQL
```